lib/spack/spack/cmd/audit.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import llnl.util.tty.color as cl
+import spack.audit
+import spack.repo
+
+
+description = "audit configuration files, packages, etc."
+section = "system"
+level = "short"
+
+CHECKS = {
+    'configs': ['CFG-COMPILER', 'CFG-PACKAGES'],
+    'packages': ['PKG-DIRECTIVES']
+}
```

Comment:
I still think it makes sense to define the group when creating the decorator:

```
audit_pkgdirectives = AuditClass(
    tag='PKG-DIRECTIVES',
    group='packages',  # <--- define the group here
    description='Sanity checks on specs used in directives',
    kwargs=('pkgs',)
)
```

Reply:
It probably still makes sense to keep the overall group names (`packages`, `configs`) here since each corresponds to specific subcommand argument processing logic. But I think the association of individual check to group should be done where the `AuditClass` is defined.

Reply:
Done in 9739b42